app/views/claims/mostly_performed_leadership_duties.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render("shared/error_summary", instance: current_claim, errored_field_id_overrides: { mostly_performed_leadership_duties: "claim_eligibility_attributes_mostly_performed_leadership_duties_true"}) if current_claim.errors.any? %>
```

Comment:
Should this be `"eligibility.mostly_performed_leadership_duties": ` so that it links to the correct ID?

Reply:
Ugh. Yes. I suspect there will be others, since this was edited from the now-deleted `mostly_teaching_eligible_subjects` view.

Reply:
I fixed this and went through and fixes the rest of the eligibility error tags, too.